sourcemash/api/categories.py

```diff
@@ -44,24 +79,13 @@ def get(self):
                              .all()
 
         for category, count in categories:
-            read_item_count = UserItem.query \
-                                .filter(UserItem.user_id==current_user.id) \
-                                .join(UserItem.item) \
-                                .filter(Item.cats.contains(category)) \
-                                .filter(~UserItem.unread) \
-                                .count()
-
-            category.item_count = count
-            category.unread_count = category.item_count - read_item_count
 
-            unsubscribed_item = Item.query.filter(Item.cats.contains(category)) \
+            if count > MIN_ITEM_COUNT:
+                unsubscribed_item = Item.query.filter(Item.cats.contains(category)) \
                                           .filter(~Item.feed_id.in_(user_feed_ids)) \
                                           .first()
-            if unsubscribed_item:
```

Comment:
Not a problem since the PUT is triggered from the frontend so it just sees it as another item in the collection.